app/services/whats_app_adapter/three_sixty_dialog/template_creator_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module WhatsAppAdapter
+  module ThreeSixtyDialog
+    class TemplateCreatorService
```

Comment:
I see you calling this always with `TemplateCreatorService.new(/*...*/).call` you can make this a simple (class) method then.